### ISSUE TITLE
Validation perplexity not decreasing below a value

### ISSUE THREAD

**Issue by shahbazsyed** [OpenNMT/OpenNMT#294]
Hi,
I trained a summarization model using 3 layers, LSTM, rnn_size=600, SGD, learning_rate = 1 for 15 epochs,with start_decay at epoch 9. The lowest validation perplexity achieved was 45. I resumed training from this epoch_15_45 for another 8 epochs with learning rate of 0.1 . However my validation perplexity did not decrease any further. 
What can be done in such scenarios to decrease the perplexity/ make the model learn further?

Can someone also kindly let me know how to use the start_decay_ppl_delta option? Is this the value 45 in my case, or the difference between perplexity values of two epochs? (for e.g say I need atleast a difference of 10 between the perplexity values, and if not, then start the decaying process )

Thanks!

**Comment by guillaumekln** [comment]
Hi,

> What can be done in such scenarios to decrease the perplexity/ make the model learn further?

With the same model, it seems you are already close to the optimal point. Otherwise, you could train a bigger model and try various [model configurations](http://opennmt.net/OpenNMT/training/models/) to push the perplexity down.

> Can someone also kindly let me know how to use the start_decay_ppl_delta option? Is this the value 45 in my case, or the difference between perplexity values of two epochs?

It's a relative value. If `previous_ppl - current_ppl < start_decay_ppl_delta` then the learning is decayed.

**Comment by shahbazsyed** [comment]
Thanks! @guillaumekln
